scripts/IA/consume_files.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import os
+import argparse
+from zipfile import ZipFile
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+	'-g',
+	'--guid',
+	help='This is the GUID of the target node on the OSF'
+)
+parser.add_argument(
+	'-d',
+	'--directory',
+	help='This is the target Directory for the project and its files'
+)
+
+def main(default_args=True):
+	if (default_args):
```

Comment:
It's ok to parse these outside of `main `down around ln 55 it makes it much easier to write unit tests when you don't have to mock the script args.